bliss/conf/base_config.yaml

```diff
@@ -17,41 +17,41 @@ paths:
     data: ${paths.root}/data
     sdss: ${paths.data}/sdss
     decals: ${paths.data}/decals
+    des: ${paths.data}/des
     dc2: ${paths.data}/dc2
     output: ${paths.root}/output
     pretrained_models: ${paths.data}/pretrained_models
 
+prior:
+    reference_band: 2  # r-band
+    star_color_model_path: ${simulator.survey.dir_path}/color_models/star_gmm_nmgy.pkl
+    gal_color_model_path: ${simulator.survey.dir_path}/color_models/gal_gmm_nmgy.pkl
+    n_tiles_h: 20
+    n_tiles_w: 20
+    tile_slen: 4
+    batch_size: 64
+    max_sources: 1
+    mean_sources: 0.2
+    min_sources: 0
+    prob_galaxy: 0.72
+    star_flux_min: 0.63094948
+    star_flux_max: 1014
+    star_flux_alpha: 0.43
+    galaxy_flux_min: 0.6301037
+    galaxy_flux_max: 1013
+    galaxy_alpha: 0.47
+    galaxy_a_concentration: 0.39330758068481686
+    galaxy_a_loc: 0.8371888967872619
+    galaxy_a_scale: 4.432725319432478
+    galaxy_a_bd_ratio: 2.0
+
```

Comment:
It's fine to define `prior` at the top-level of the yaml, but it should still get passed as argument to simulator (i.e. `prior: ${prior}`, under `simulator:`)

Reply:
Done.